config/connection.js

```diff
@@ -0,0 +1,18 @@
+// require("dotenv").config();
```

Comment:
You have commented out all of your connection file so there is no way to connect to the database. You are also missing your seed files to seed the database.